config/routes.php

```diff
@@ -67,7 +67,7 @@
         [
             'controller' => 'Listings',
             'action' => 'list',
-            'sc21',
+            'sc22',
```

Comment:
I guess this is retroactively fixing the issue I pointed out in the first patch.